src/core/executors/SubscriptionExecutor.ts

```diff
@@ -28,7 +28,7 @@ export class SubscriptionExecutor<Model> extends ExecutorBase<Model> {
   }
 
   private separateDeltasByChangeType(deltas: CoreDelta<Model>[]): { [key: string]: CoreDelta<Model>[] } {
-    const deltasByChangeType: {[key in CoreChangeType]: CoreDelta<Model>[]} = {
+    const deltasByChangeType: Partial<{[key in CoreChangeType]: CoreDelta<Model>[]}> = {
```

Comment:
Explore creating a new base type and extend that to include the hydrator change type

Reply:
Will leave as is. Typescript offers no language feature to extend enums.